.github/workflows/prod-cf-deploy-lambda.yml

```diff
@@ -0,0 +1,28 @@
+### This is the Terraform-generated GitHub Actions workflow for cf-lambda    ###
+### applications for dev/stage/prod. It will publish the Lambda zip to S3    ###
+### and automatically re-apply Terraform in the Terraform Cloud              ###
+### workloads-libraries-website-prod workspace.                              ###
```

Comment:
Can you confirm these docs are correct? `TF_AUTO_APPLY: false` doesn't fit my brain with these words.

Reply:
The stuff in the ###    ### block are the comments that are generated by a single template file in the Terraform repo. What you discovered here is the fact that I didn't not edit that text properly in that comment! The PROD workflow does not auto-apply Terraform in Terraform Cloud -- it runs a `terraform apply` command, but waits for human approval before applying it.

I will update the docs. For now, we are not trying to do any notification yet from the prod GHA action, that will come in a future feature branch...